pom.xml

```diff
@@ -66,7 +66,7 @@
         <suite.domain.default.profile>full</suite.domain.default.profile>
         <suite.domain.default.server>server-one</suite.domain.default.server>
         <suite.mode/>
-        <suite.url>http://localhost:9990/console/ts.html</suite.url>
+        <suite.url>http://localhost:9990/console/index.html</suite.url>
```

Comment:
What's the motivation for switching from `ts.html` to `index.html`?

Reply:
It probably got included when trying to fix tests that were caused by loading page timeouts. I'll revert that

Reply:
Fixed with d4fa84ed488931f0482f81e7be7a316a2943fc86

Reply:
Strange, GitHub seems malfunctioning here. It is indeed fixed in d4fa84e.